website/blog/2020-05-25-7.10.0.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "7.10.0 Released: Class Fields in preset-env, '#private in' checks and better React tree-shaking"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2020-05-25 0:00:00
+categories: announcements
+share_text: "Babel 7.10.0 Released"
+---
+
+We just released a new minor version of Babel!
+
+This 7.10 release includes:
+- Full support for the new Stage 1 proposal, `#prop in obj` checks for private fields [proposal](https://github.com/tc39/proposal-private-fields-in-in) (`#prop in obj`).
```

Comment:
The second `(`#prop in obj`)` seems redundant?

Reply:
```suggestion
- Full support for the new Stage 1 proposal, `#prop in obj` checks for private fields [proposal](https://github.com/tc39/proposal-private-fields-in-in).
```